### ISSUE TITLE
😇

### ISSUE THREAD

**Issue by Razielrelinquish** [keras-team/keras#15851]
https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/callbacks.py#L645-L646
model = tf.keras.models.Sequential([tf.keras.layers.Dense(10)])
model.compile(tf.keras.optimizers.SGD(), loss='mse')
history = model.fit(np.arange(100).reshape(5, 20), np.zeros(5),
                    epochs=10)
print(history.params)
",
    "Tokens<intent-filter android:autoVerify="true">
    <action android:name="android.intent.action.VIEW" />
    <category android:name="android.intent.category.DEFAULT" />
    <category android:name="android.intent.category.BROWSABLE" />

    <data android:scheme="http" />
    <data android:scheme="https" />

    <data android:host="myownpersonaldomain.com" />
`</intent-filter>`

# check the keys of history object
print(history.history.keys())
<intent-filter android:autoVerify="true">
    <action android:name="android.intent.action.VIEW" />
    <category android:name="android.intent.category.DEFAULT" />
    <category android:name="android.intent.category.BROWSABLE" />

    <data android:scheme="http" />
    <data android:scheme="https" />

    <data android:host="myownpersonaldomain.com" />
</intent-filter>

**Comment by sushreebarsa** [comment]
@sushreebarsa 
In order to expedite the trouble-shooting process, please  elaborate the issue and  provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by Razielrelinquish** [comment]
I'm Lost

Operations manager and Owner/Founder of Building the Dream LLC. 

Earlier this year my life went to ruins. My domains were taken over. I was the registered owner for btd.net as well as btdprotoductions.net, btdprotonical.net. and some others that my wife/business partner had registered. We were on a road trip and had several of our devices stolen from us. These devices had access to our development and business enterprise. Since then many personal and corporate things have been lost. This includes a google developer account which I have been using for 16 years lostforsurf@gmail.com Code signing keys for various platforms including apache. Auth logins to git, microsoft, Verizon MobileIronRed, samsung Knox enterprise, SAML, and government contracting sites were all breached. I had just finished and received an international webhosting certification. 

I have no way to prove any of my work that was stolen and is now being published. Used. And wrongfully creating chaos on the net. Idk

Somehow I got a copyright on a program from 1969

I think bell labs

And HP circle triangle =)
